servers/Azure.Mcp.Server/README.md

```diff
@@ -1,150 +1,298 @@
-
-[![Install Azure MCP in VS Code](https://img.shields.io/badge/VS_Code-Install_Azure_MCP_Server-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://vscode.dev/redirect?url=vscode:extension/ms-azuretools.vscode-azure-mcp-server) [![Install Azure MCP in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_Azure_MCP_Server-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://vscode.dev/redirect?url=vscode-insiders:extension/ms-azuretools.vscode-azure-mcp-server) [![Install Azure MCP in Visual Studio](https://img.shields.io/badge/Visual_Studio-Install_Azure_MCP_Server-C16FDE?style=flat-square&logo=visualstudio&logoColor=white)](https://marketplace.visualstudio.com/items?itemName=github-copilot-azure.GitHubCopilotForAzure2022)
+This `README.md` file serves as a shared template for multiple package repositories. To generate a package-specific README, use special comment annotations to mark sections of the markdown that should be conditionally, excluded, or replaced. The script `eng\scripts\Process-PackageReadMe.ps1` automates this processing.
 
+## Usage
 
-## Table of Contents
-- [Overview](#overview)
-- [Installation](#installation)
-    - [IDE Extensions](#ide-extensions) 
-        - [VS Code (Recommended)](#vs-code-recommended)
-        - [Visual Studio 2022](#visual-studio-2022)
-        - [IntelliJ IDEA](#intellij-idea)
-    - [Package Managers](#package-managers) 
-        - [NuGet](#nuget)
-        - [NPM](#npm)
-        - [Docker](#docker)
-    - [Custom Clients](#custom-clients)
-- [Usage](#usage)
-    - [Getting Started](#getting-started)
-    - [What can you do with the Azure MCP Server?](#what-can-you-do-with-the-azure-mcp-server)
-    - [Complete List of Supported Azure Services](#complete-list-of-supported-azure-services)
-- [Support & Reference](#support-and-reference)
-    - [Documentation](#documentation)
-    - [Feedback & Support](#feedback-and-support)
-    - [Security](#security)
-    - [Data Collection](#data-collection)
-    - [Contributing & Code of Conduct](#contributing)
+To generate a README tailored for a specific package type, run:
 
-# <a id="overview"></a> Overview
+eng\scripts\Process-PackageReadMe.ps1 -InputReadMePath "<path to this readme>" -OutputDirectory "<output directory>" -PackageType "<nuget|npm|vsix>"
 
-**Azure MCP Server** supercharges your agents with Azure context across **30+ different Azure services**.
+## Supported Annotations
 
-# <a id="installation"></a> Installation
+All annotations must be wrapped in markdown comments . The following annotations are supported:
 
-## <a id="ide-extensions"></a> 🧩 IDE Extensions
+### 1. Section Removal
 
-Follow these simple steps to start using Azure MCP with your favorite IDE.  We recommend VS Code:
+**Purpose:** Remove one or more lines, or parts of a line of markdown for specified package types.
 
-### <a id="vs-code-recommended"></a> 🔷 VS Code (Recommended)
+**Syntax:**
```

Comment:
I think we should put this syntax documentation in another readme or in the PS script itself.